### ISSUE TITLE
Half a second video then loading then actual video

### ISSUE THREAD

**Issue by FMisplon** [yuliskov/SmartYouTubeTV#255]
When i start a video the video starts for half a second then i see loading and half a second later the video plays (totally)

**Comment by yuliskov** [comment]
This is intentional behaviour, if I understood you correctly. On some devices video decoder still not initialized when first frame arrived. So, as a workaround, I do pause and start video again.

**Comment by rodalpho** [comment]
Yes, I have the same problem. I assumed it was because that video tried to play a pre-roll ad. It's annoying but not a huge issue. Perhaps delaying playback by 250ms or so might help?

**Comment by FMisplon** [comment]
Not a huge issue but indeed a little bit annoying.  Running the app on a shield so not exactly on a slow device...
